### PR TITLE
[organization] Associate OICD-created User with Organization

### PR DESCRIPTION
**Pull request by geropl**
## Related Issue(s)

Fixes #

## Release Notes

```release-note
Associate users logging in with a SSO provider to that Organization (and make the first an owner)
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`
- [ ] /werft publish-to-npm

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-dedicated-emulation
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by svenefftinge — APPROVED** [review]
code LGTM

File: components/gitpod-db/src/typeorm/migration/1675433412120-UserOrganizationId.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists, indexExists } from "./helper/helper";
+
+const table = "d_b_user";
+const column = "organizationId";
+const orgIndex = "idx_organizationId";
+
+export class UserOrganizationId1675433412120 implements MigrationInterface {
```
Comment (easyCZ) [inline components/gitpod-db/src/typeorm/migration/1675433412120-UserOrganizationId.ts:14]: In this PR, there's a mix of schema changes, and applications using it. In a meeting on Tue Jan 31st, we agreed we wouldn't lump schema and application changes together as it prevents us from ensuring that schema changes are always applied before application changes which use those - crucial for Dedicated.

Is it possible to split this into just the schema update, which is backwards compatible, and then the application changes on top?

**Comment by geropl** [comment]
Did not add hold before chaning the PR status :see_no_evil: Was still writing the test instructions... :smile: 

> Is it possible to split this into just the schema update, which is backwards compatible, and then the application changes on top?

That was the plan, just wanted to have one PR for testing.

That being said, in this particular case it does not affect code that is run by any user on gitpod.io, yet, because we guard it with a FeatureFlag. :+1:

**Comment by svenefftinge** [comment]
Did you apply the change on gitpod.io already?

**Comment by geropl** [comment]
No, but I would to that now.

**Pull request merged**